Readme.md

```diff
@@ -17,11 +17,14 @@ A react-native wrapper for handling in-app purchases.
 
 ### Add it to your project
 
-1. Make sure you have `rnpm` installed: `npm install rnpm -g`
+1. Install and link the react-native-in-app-utils package:
 
-2. Install with rnpm: `rnpm install react-native-in-app-utils`
+```
+react-native install react-native-in-app-utils
```

Comment:
If `npm install` is used, `react-native link` would also need to be used. (See https://stackoverflow.com/a/43728793/4284484)